src/compton_tools/python/common_compton.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+# --------------------------------------*-python-*------------------------------------------------ #
+# file   src/compton_tools/python/common_compton.py
+# author Andrew Till <till@lanl.gov>
+# date   14 May 2020
+# brief  This script has functions that parse an ULTRA file with Compton
+#        data and return a dense matrix and energy/temperature grids;
+#        If run as executable, saves grids and data with same base filename
+# note   Copyright (C) 2020, Triad National Security, LLC. All rights reserved.
+# ------------------------------------------------------------------------------------------------ #
+
+# ------------------------------------------------------------------------------------------------ #
+# TPL
+import numpy as np
+# ------------------------------------------------------------------------------------------------ #
+
+# These are the common functions that are used to read and write Compton data
+
+
+# ------------------------------------------------------------------------------------------------ #
+def print_grids(grids, fileroot, verbosity=False):
```

Comment:
@alexrlongne Can you review the python here?

Reply:
For reference, all of the python in `src/compton_tools/python` are little scripts intended for internal use so we don't have to keep rewriting our own python readers of Compton data

Reply:
Looks good, just the small comment about the normalization constant.